releases.md

```diff
@@ -42,10 +42,10 @@ Further documentation available:
 
 ### v0.22
 
-- **Latest Release**: [v0.22.1][v0-22-1] (2023-01-13) ([docs][v0-22-1-docs], [examples][v0-22-1-examples])
-- **Initial Release**: [v0.22.0][v0-22-0] (2022-11-16)
+- **Latest Release**: [v0.22.2][v0-22-2] (2023-02-21) ([docs][v0-22-2-docs], [examples][v0-22-2-examples])
+- **Initial Release**: [v0.22.1][v0-22-1] (2023-01-13)
```

Comment:
The initial release is v0.22.0

Reply:
ah okay if we take that way yes its 22.0

Reply:
updated